qdrant-landing/content/documentation/headless/snippets/text-search/batch-progressive-filter/generated/go.md

```diff
@@ -2,31 +2,31 @@
 strict := &qdrant.QueryPoints{
 	CollectionName: "books",
 	Query: qdrant.NewQueryNearest(
-		qdrant.NewVectorInputDocument(&qdrant.Document{ Model: "sentence-transformers/all-minilm-l6-v2", Text: "time travel" }),
+		qdrant.NewVectorInputDocument(&qdrant.Document{Model: "sentence-transformers/all-minilm-l6-v2", Text: "time travel"}),
 	),
-	Using: qdrant.PtrOf("description-dense"),
-	Filter: &qdrant.Filter{ Must: []*qdrant.Condition{ qdrant.NewExpressionCondition(qdrant.NewMatchText("title", "time travel")) } },
```

Comment:
Before https://github.com/qdrant/landing_page/pull/2056/commits/f010bdb70a12b19623889c41ec95b2ec9c77b7cc, the check snippets CI should have failed. There is a type error.

cc @xzfc